modules/education/src/main/java/edu/stanford/spezi/modules/education/videos/component/ExpandableVideoSection.kt

```diff
@@ -156,13 +154,13 @@ private fun VideoItem(video: Video, onVideoClick: () -> Unit) {
                 .padding(Spacings.small)
                 .fillMaxWidth()
         ) {
-            AsyncImageResourceComposable(
-                imageResource = AsyncImageResource.Remote(url = video.thumbnailUrl, StringResource("Video thumbnail")),
+            AsyncImageResource(
```

Comment:
Don't you need to call Content on the resource then? I'd assume that the resource itself isn't a composable?